config/rbac/role.yaml

```diff
@@ -12,94 +12,12 @@ rules:
   verbs:
   - create
   - patch
-- apiGroups:
-  - ""
```

Comment:
Why are we removing this?

Reply:
Not really removing. This is auto-generated by controller-gen, and, after it's been updated, the new version consolidated rbac permissions into fewer sections. I.e. in the new file https://github.com/digitalocean/do-operator/blob/976400bfda72e67b2174408058342d28b200bffd/config/rbac/role.yaml there's still  create and patch on secrets, it's just bundled together with configmaps. Similar with database resources - they're just bundled into a single section with the same verbs